src/DataObjects/QueuedJobDescriptor.php

```diff
@@ -585,4 +611,17 @@ private function getWorkerExpiry()
 
         return $time->getTimestamp() - $now->getTimestamp();
     }
+
+    public function getDefaultSearchContext()
+    {
+        $context =  parent::getDefaultSearchContext();
+
+        $context->addField($this->buildJobStatusField()->setEmptyString(''));
+        $context->addFilter(ExactMatchFilter::create('JobStatus'));
```

Comment:
You can define filters in the searchable_fields array, which would probably be preferable here.
https://docs.silverstripe.org/en/4/developer_guides/model/scaffolding/#specify-a-form-field-or-search-filter

Reply:
Done